course_catalog/models.py

```diff
@@ -312,6 +313,14 @@ def certification(self):
         else:
             return []
 
+    @property
+    def level(self):
+        """Returns the level for the course"""
+        if self.runs.last():
+            return self.runs.last().level
```

Comment:
Some runs for a course have different levels, for example:
```
Course Title:Topics in Philosophy of Mind: Mental Content
	 Run Title: Other Minds, Level: Undergraduate, URL: http://ocw.mit.edu/courses/linguistics-and-philosophy/24-500-other-minds-spring-2003
	 Run Title: Topics in Philosophy of Mind: Mental Content, Level: Graduate, URL: http://ocw.mit.edu/courses/linguistics-and-philosophy/24-500-topics-in-philosophy-of-mind-mental-content-spring-2015
	 Run Title: Topics in Philosophy of Mind: Perceptual Experience, Level: Graduate, URL: http://ocw.mit.edu/courses/linguistics-and-philosophy/24-500-topics-in-philosophy-of-mind-perceptual-experience-spring-2007
	 Run Title: Topics in Philosophy of Mind: Self-Knowledge, Level: Graduate, URL: http://ocw.mit.edu/courses/linguistics-and-philosophy/24-500-topics-in-philosophy-of-mind-self-knowledge-spring-2005
Course Title:Making Science and Engineering Pictures: A Practical Guide to Presenting Your Work
	 Run Title: Making Science and Engineering Pictures: A Practical Guide to Presenting Your Work, Level: Undergraduate, URL: http://ocw.mit.edu/resources/res-10-001-making-science-and-engineering-pictures-a-practical-guide-to-presenting-your-work-spring-2016
	 Run Title: Making Science and Engineering Pictures, Level: Graduate, URL: http://ocw.mit.edu/resources/res-10-001-making-science-and-engineering-pictures-fall-2014
Course Title:Mechatronics
	 Run Title: Mechatronics, Level: Undergraduate, URL: http://ocw.mit.edu/courses/mechanical-engineering/2-737-mechatronics-spring-1999
	 Run Title: Mechatronics, Level: Graduate, URL: http://ocw.mit.edu/courses/mechanical-engineering/2-737-mechatronics-fall-2014
```

So maybe the search facet should use the run levels directly instead of adding a course level? The former availability facet used the run dates for example.

Reply:
Sounds good. I removed level from the course index